src/main/resources/sql/genericVisitReport.sql

```diff
@@ -30,7 +30,7 @@ SELECT name FROM patient_identifier_type WHERE uuid = @primaryIdentifierTypeUuid
 SET @sql = CONCAT('SELECT
   GROUP_CONCAT(DISTINCT(IF(pit.name = @primaryIdentifierTypeName, pi.identifier, NULL)))     AS "Patient Identifier",
   ',IF(@extraPatientIdentifierTypes = '', '', CONCAT(@extraPatientIdentifierTypes, ',')),'
-  concat(pn.given_name, " ", ifnull(pn.family_name, ""))                    AS "Patient Name",
+  concat(pn.family_name, " ", ifnull(pn.given_name, ""))                    AS "Patient Name",
```

Comment:
@rubailly `ifnull` needs also to move with family_name
```
concat(ifnull(pn.family_name, ""), " ", pn.given_name)
```

Reply:
Done. ..thanks @tmvumbi2 !